Flutter/DateRangePicker/hijri-date-picker.md

```diff
@@ -15,7 +15,7 @@ It consists of all the Gregorian calendar functionalities like min and max date,
 To display the Hijri date picker, initialize the [HijriDateRangePicker](https://pub.dev/documentation/syncfusion_flutter_datepicker/latest/datepicker/SfHijriDateRangePicker-class.html) widget as a child of any widget. Here, the Hijri date range picker added as a child of the scaffold widget.
 
 {% tabs %}
-{% highlight Dart %}
+{% highlight dart hl_lines="4" %}
```

Comment:
Missing highlight on https://help.syncfusion.com/flutter/daterangepicker/builders#cell-builder

Reply:
highlighted the mentioned topics.

Reply:
64 line removed.